site/content/contribute/mobile/build-your-own.md

```diff
@@ -174,9 +174,9 @@ changes in the project.
 
 - Replace `android/app/google-services.json` with the one you downloaded earlier
 - Open `android/app/google-services.json`, find the project\_number and copy the value
-- Open `android/app/src/main/AndroidManifest.xml` file, look for the line `<meta-data android:name="com.wix.reactnativenotifications.gcmSenderId" android:value="184930218130\0"/>` and replace the value with the one that you copied in the previous step
+- Open `android/app/src/main/AndroidManifest.xml` file, look for the line `<meta-data android:name="com.wix.reactnativenotifications.gcmSenderId" android:value="184930218130\"/>` and replace the value with the one that you copied in the previous step
 
-**Leave the trailing \0 intact**
+**Leave the trailing \ intact**
```

Comment:
Perhaps I misunderstand, but shouldn't we remove all of `\0` and not just the `0`, additionally removing this comment about keeping both together? I can't see how `...android:value="184930218130\"/>` is semantically correct.

Reply:
Really, though, this is a question for @enahum, since the docs here match the actual changes in https://github.com/mattermost/mattermost-mobile/pull/2169/files.

Reply:
^ also, @saturninoabril

Reply:
This is exactly how it should be, the ID followed by a \ and nothing more

Reply:
Ok! Thanks for confirming :)